Update Get-CsOnlineUser.md (#10159)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsOnlineUser.md

```
Example 6 returns information for user accounts that are categorized as resource accounts. 

### -------------------------- Example 7 --------------------------
```

> Get-CsOnlineUser - Filter "FeatureTypes -Contains 'PhoneSystem'"

Reviewer: ```suggestion
Get-CsOnlineUser -Filter "FeatureTypes -Contains 'PhoneSystem'"
```

```
Example 7 returns information for user's assigned plans. 

> 

Reviewer 2: ```suggestion
```

- PhoneSystem
- CallingPlan

> **NOTE:** This attribute is now filterable in Teams Powershell Module versions 4.0.0 and later using the "-Contains" operator as shown in Example 7

Reviewer 2: ```suggestion
**Note**: This attribute is now filterable in Teams Powershell Module versions 4.0.0 and later using the "-Contains" operator as shown in Example 7.
```

> **Note**: This attribute is now filterable in Teams Powershell Module versions 4.0.0 and later using the "-Contains" operator as shown in Example 7.

Reviewer 2: ```suggestion
**Note**: This attribute is now filterable in Teams PowerShell Module versions 4.0.0 and later using the "-Contains" operator as shown in Example 7.
```

- PhoneSystem
- CallingPlan

> _Note -_ This attribute is now filterable in Teams Powershell Module versions 4.0.0 and above using the "-Contains" operator as showon in Example 7

Reviewer 2: ```suggestion
**NOTE:** This attribute is now filterable in Teams Powershell Module versions 4.0.0 and later using the "-Contains" operator as shown in Example 7
```
